ArticleTemplates/assets/js/modules/comments.js

```diff
@@ -26,8 +26,8 @@ define([
                                 var numOfComments = $(this)[0].children.length - 4;
                                 if (numOfComments == 1) {
                                     $(this).addClass("block--discussion-thread--orphan");
-                                } else {
-                                    $(blockID).after("<div class='more more--comments'><a class='more__label'><span data-icon='&#xe050;' aria-hidden='true'></span>" + numOfComments + " more replies</a></div>");
+                                } else {                            
```

Comment:
maybe something you can change in your editor settings to tidy up trailing whitespace on save?